src/bioregistry/data/curated_papers.tsv

```diff
@@ -0,0 +1,21 @@
+pmid	relevant	relevancy_type	notes	pr_added	orcid	date_curated
+39104285	1	new_provider	Provider for UniProt IDs, issue with curation due to multiple URI formats depending on plant species	1193	0009-0009-5240-7463	2024-09-24
+39074139	1	new_provider	Resolver for PDB IDs	1193	0009-0009-5240-7463	2024-09-24
+39014503	0	no_website		1193	0009-0009-5240-7463	2024-09-25
+39047988	0	irrelevant_other			0009-0009-5240-7463	2024-09-25
+39115390	0	irrelevant_other			0009-0009-5240-7463	2024-09-26
+39095357	0	irrelevant_other			0009-0009-5240-7463	2024-09-26
+39084442	0	not_identifiers_resource			0009-0009-5240-7463	2024-09-27
+38991851	1	unclear	identifiers for sharing, retrieving, and validating sample metadata. Unclear if this should be curated as a prefix, provider or a separate registry		0009-0009-5240-7463	2024-09-28
+38991828	0	irrelevant_other			0009-0009-5240-7463	2024-09-28
+39049520	0	not_identifiers_resource			0009-0009-5240-7463	2024-09-30
+39104826	1	existing	Already present in the bioregistry as a provider for mesh prefix		0009-0009-5240-7463	2024-10-01
+39050757	0	irrelevant_other			0009-0009-5240-7463	2024-10-01
+39064021	0	irrelevant_other			0009-0009-5240-7463	2024-10-01
+39028894	0	not_identifiers_resource			0009-0009-5240-7463	2024-10-04
+39044201	0	not_identifiers_resource	Potential resource for rare diseases identifiers, but not identifier information		0009-0009-5240-7463	2024-10-04
+39088253	0	irrelevant_other			0009-0009-5240-7463	2024-10-05
+39119155	0	irrelevant_other			0009-0009-5240-7463	2024-10-05
+39005357	0	irrelevant_other			0009-0009-5240-7463	2024-10-05
+39044130	0	irrelevant_other			0009-0009-5240-7463	2024-10-05
+39010878	0	irrelevant_other			0009-0009-5240-7463	2024-10-05
```

Comment:
@nagutm One final request here: why don't we make the `notes` column last since it has variable length, to make the file a bit easier to read.